packages/jest-emotion/src/index.js

```diff
@@ -37,12 +39,14 @@ function getClassNamesFromDOMElement(selectors, node) {
   return getClassNames(selectors, node.getAttribute('class'))
 }
 
-function getClassNamesFromNodes(nodes) {
+export function getClassNamesFromNodes(nodes) {
```

Comment:
Could you move this into a utils file and import it from that file because exporting from here means it’s exported from the main package so people could start using it and if we wanted to change it we would have to do a major version